operators/go/submariner-operator/.gitignore

```diff
@@ -0,0 +1,77 @@
+# Temporary Build Files
+build/_output
+build/_test
+# Created by https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
```

Comment:
Such a huge file is unnecessary, if you're using a specific IDE you should have that ignore locally on your system, not checked in.

Reply:
That's automatically generated by operator-sdk, let's keep it.

Reply:
Right, it's the operator-sdk default, nothing I did.